### PR TITLE
sqlite 3.8.10

### PR DESCRIPTION
**Pull request by vszakats**

### CODE REVIEW

File: Library/Formula/sqlite.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@ class Sqlite < Formula
   option "with-unlock-notify", "Enable the unlock notification feature"
   option "with-icu4c", "Enable the ICU module"
   option "with-functions", "Enable more math and string functions for SQL queries"
+  option "with-dbstat", "Enable the \"dbstat\" virtual table"
```
Comment (DomT4) [inline Library/Formula/sqlite.rb]: Why `\"dbstat\"` instead of `Enable the 'dbstat' virtual table`?

Comment (vszakats) [inline Library/Formula/sqlite.rb]: No strong reason, I copied the sentence from the sqlite change listing and kept the double-quotes. I'll update to single ones.

**Comment by tdsmith** [comment]
Can you squash these to a single commit please?

**Comment by xu-cheng** [comment]
Thanks.

**Pull request closed without merging**